runtime/mandala/src/lib.rs

```diff
@@ -1451,8 +1478,8 @@ parameter_types! {
 
 #[cfg(not(feature = "with-ethereum-compatibility"))]
 parameter_types! {
-	pub NativeTokenExistentialDeposit: Balance = microcent(ACA);
-	pub const NewContractExtraBytes: u32 = 10_000;
+	pub NativeTokenExistentialDeposit: Balance = 10 * cent(ACA);
+	pub const NewContractExtraBytes: u32 = 10_000_000;
```

Comment:
Here changes the storage limit.
https://github.com/AcalaNetwork/Acala/blob/9870c765d133d64cc350789b78e3f098b27eaba3/modules/evm/src/lib.rs#L1052-L1053

@xlc Maybe need to add `gas_limit` and `storage_limit` arguments for `set_code` extrinsic.